packages/block-editor/src/components/font-appearance-control/index.js

```diff
@@ -39,6 +39,8 @@ const getFontAppearanceLabel = ( hasFontStyles, hasFontWeights ) => {
  */
 export default function FontAppearanceControl( props ) {
 	const {
+		/** Start opting into the larger default height that will become the default size in a future version. */
+		__next40pxDefaultSize = false,
```

Comment:
Should we add  `FontAppearanceControl` to the lint rule too? Or at least a TODO comment to update this in the future?

Reply:
Hmm, good point 🤔 I'll go back and batch add the affected wp-block-editor components.

Reply:
Addressed in #64591